pages/docs/user-docs/docs-bootstrap.md

```diff
@@ -25,24 +25,30 @@ The YUM bootstrap module uses YUM on the host system to bootstrap the core opera
 
-##### **debootstrap**
+#### **debootstrap**
 The Debian bootstrap module is a tool which is used specifically for bootstrapping distributions which utilize the `.deb` package format and `apt-get` repositories. This module will bootstrap any of the Debian and Ubuntu based distributions. When using the `debootstrap` module, the following keywords must also be defined:
 
  - **MirrorURL**: This is the location where the packages will be downloaded from. When bootstrapping different Debian based distributions of Linux, this will define which varient will be used (e.g. specifying a different URL can be the difference between Debian or Ubuntu).
  - **OSVersion**: This keyword must be defined as the alpha-character string associated with the version of the distribution you wish to use. For example, `trusty` or `stable`. 
  - **Include**: As with the `yum` module, the `Include` keyword will install additional packages into the core operating system and the best practice is to supply only the bare essentials such that the `%inside` scriptlet has what it needs to properly completely the bootstrap.
 
-##### **arch**
+#### **arch**
 The Arch Linux bootstrap module does not name any additional keywords at this time. By defining the `arch` module, you have essentially given all of the information necessary for that particular bootstrap module to build a core operating system.
 
-##### **docker**
+#### **docker**
 The Docker bootstrap module will create a core operating system image based on an image hosted on a particular Docker Registry server. By default it will use the primary Docker Library, but that can be overridden. When using the `docker` module, several other keywords may also be defined:
 
  - **From**: This keyword defines the string of the registry name used for this image in the format [name]:[version]. Several examples are: `ubuntu:latest`, `centos:6`, `alpine:latest`, or `debian` (if the version tag is ommitted, `:latest` is automatically used).
  - **IncludeCmd**: This keyword tells Singularity to utilize the Docker defined `Cmd` as the `%runscript` (defined below), if the `Cmd` is defined.
  - **Registry**: If the registry you wish to download the image from is not from the main Docker Library, you can define it here.
  - **Token**: Sometimes the Docker API (depending on version?) requires an authorization token which is generated on the fly. Toggle this with a `yes` or `no` here.
 
+#### **localimage**
+This module allows you to bootstrap a container from an existing Singularity image on your host system.  You can use an existing container image as your "base" and then customize it with a new `%post` `%environment`, `%runscript`, etc.  
+
```

Comment:
I would add a note here to hint about why they would want to do this. Eg:

>> This module allows you to bootstrap a container from an existing Singularity image on your host system.  You can use an existing container image as your "base," and then add customization to when building a new container. For example, I might have bases for each of the common operating systems, or different versions of python that I like to use, and then instead of building from scratch each time, I would start with the base. For each new container that I build using a base, I then get to customize the new `%post` `%environment`, `%runscript`, etc.